DuckDuckGo/Subscription/Views/SubscriptionSettingsView.swift

```diff
@@ -92,6 +98,10 @@ struct SubscriptionSettingsView: View {
                     })
                 }
             }
+            
+            NavigationLink(destination: SubscriptionGoogleView(), isActive: $viewModel.shouldDisplayGoogleView) {
+                EmptyView()
+            }
```

Comment:
Some testing leftover?

Reply:
This link is used for Google, but it needs to be moved outside the list.   It's done in a next PR